api/yelp_beans/templates/weekly_opt_in_email.html

```diff
@@ -4,8 +4,8 @@
     <td>
         <p>Hey {{first_name}},</p>
         <p>
-            Would you like to meet someone new this week? Your meetings are on {{meeting_day}}
-            at {{meeting_time}} in the {{office}} at the {{location}}.
+            Would you like to meet someone new this week in {{meeting_title}}? This meeting is officially on {{meeting_day}}
```

Comment:
Is `meeting_title` included as a variable to this template? Looking at [where I think it gets called](https://github.com/Yelp/beans/blob/master/api/yelp_beans/send_email.py#L100) it doesn't look like we pass that in.

Reply:
Nope. I'll remove that, but we should consider adding it in a future update so people can tell which subscription they're signing up for.